### PR TITLE
fix(preview): upgrade vue repl version

### PR DESCRIPTION
**Pull request by chilingling**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe:  upgrade dependencies vue/repl

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: packages/design-core/src/preview/src/preview/Preview.vue [hunk 1]
```diff
@@ -119,7 +110,7 @@ export default {
 
     const params = getSearchParams()
 
-    const promiseList = [fetchCode(params), fetchMetaData(params), setFiles(files, 'Main.vue')]
+    const promiseList = [fetchCode(params), fetchMetaData(params), setFiles(srcFiles, 'src/Main.vue')]
     Promise.all(promiseList).then(([codeList, metaData]) => {
```
Comment (kevinmoch) [inline packages/design-core/src/preview/src/preview/Preview.vue:115]: 在异步函数中最好使用 try-catch 来处理异步操作可能抛出的错误。尤其是在 Promise.all 中，如果其中一个 Promise 被 reject，整个 Promise.all 也会被 reject。

File: packages/design-core/src/preview/src/preview/Preview.vue [hunk 2]
```diff
@@ -90,22 +92,11 @@ export default {
...
-      const files = {}
-      Object.entries(newFiles).forEach(([fileName, fileCode]) => {
-        files[fileName] = new File(fileName, fileCode)
-      })
-      const compilingList = Object.values(files).map((file) => compileFile(store, file))
-
-      await Promise.all(compilingList)
-      store.state.mainFile = mainfileName
-      store.state.files = files
+    const setFiles = async (newFiles, mainFileName) => {
```
Comment (kevinmoch) [inline packages/design-core/src/preview/src/preview/Preview.vue:95]: setFiles 函数并没有返回一个 Promise，这可能导致在 Promise.all 中无法正确等待其完成。建议将 setFiles 改为返回一个 Promise
```javascript
const setFiles = async (newFiles, mainFileName) => {
  await store.setFiles(newFiles, mainFileName);
  store.state.resetFlip = !store.state.resetFlip;
}
```

File: packages/design-core/src/preview/src/preview/Preview.vue [hunk 3]
```diff
@@ -119,7 +110,7 @@ export default {
 
     const params = getSearchParams()
```
Comment (kevinmoch) [inline packages/design-core/src/preview/src/preview/Preview.vue]: 变量 params 的命名与其他变量和函数采用了不同的命名风格。在整个代码中保持一致的命名风格有助于提高可读性。
```javascript
// 建议统一命名为 camelCase 风格
const queryParams = getSearchParams();
const promiseList = [fetchCode(queryParams), fetchMetaData(queryParams), setFiles(srcFiles, 'src/Main.vue')];
```

File: packages/design-core/src/preview/src/preview/Preview.vue [hunk 4]
```diff
@@ -90,22 +92,11 @@ export default {
...
-        files[fileName] = new File(fileName, fileCode)
-      })
-      const compilingList = Object.values(files).map((file) => compileFile(store, file))
-
-      await Promise.all(compilingList)
-      store.state.mainFile = mainfileName
-      store.state.files = files
+    const setFiles = async (newFiles, mainFileName) => {
+      store.setFiles(newFiles, mainFileName)
       store.state.resetFlip = !store.state.resetFlip
```
Comment (kevinmoch) [inline packages/design-core/src/preview/src/preview/Preview.vue:98]: 此处 store.state.resetFlip 不太容易理解，建议添加注释

File: packages/design-core/src/preview/src/preview/Preview.vue (hunk 1 above)
Comment (chilingling) [reply packages/design-core/src/preview/src/preview/Preview.vue:115]: 这里 `try catch` 之后， 需要添加出错之后的提示，需要和UX一起优化编译报错的提示

File: packages/design-core/src/preview/src/preview/Preview.vue (hunk 2 above)
Comment (chilingling) [reply packages/design-core/src/preview/src/preview/Preview.vue:95]: Done

File: packages/design-core/src/preview/src/preview/Preview.vue (hunk 3 above)
Comment (chilingling) [reply packages/design-core/src/preview/src/preview/Preview.vue]: Done

File: packages/design-core/src/preview/src/preview/Preview.vue (hunk 4 above)
Comment (chilingling) [reply packages/design-core/src/preview/src/preview/Preview.vue:98]: Done

**Review by kevinmoch — APPROVED** [review]

**Pull request merged**